src/components/CompareResults/TableRevisionContent.tsx

```diff
@@ -3,36 +3,49 @@ import { style } from 'typestyle';
 import type { compareView, compareOverTimeView } from '../../common/constants';
 import { Spacing } from '../../styles';
 import type { CompareResultsItem } from '../../types/state';
-import RevisionHeader from './RevisionHeader';
+import LinkToRevision from './LinkToRevision';
 import RevisionRow from './RevisionRow';
+import TestHeader from './TestHeader';
 
+// We're using typestyle styles on purpose, to avoid the performance impact of
+// MUI's sx prop for these numerous elements.
 const styles = {
-  tableBody: style({
-    marginTop: Spacing.Large,
+  testBlock: style({
+    /* Note that this margin will be merged with the margin below */
+    marginTop: Spacing.xLarge,
   }),
+  revisionBlock: style({ marginBottom: Spacing.Large }),
 };
 
 function TableRevisionContent(props: Props) {
   const { results, view, rowGridTemplateColumns } = props;
+  // All results are for the same test, so any results hould generate the same
```

Comment:
hould?

Reply:
The `s` at the end of `results` is the one that should be at the start of `hould` :-) fixing!